<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmKinectMain</class>
 <widget class="QMainWindow" name="FrmKinectMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Face Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/resources.qrc">:/images/logo.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnIdentify">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&amp;Identify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVerify">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&amp;Verify</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Decision threshold</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QwtSlider" name="sliderRaw">
             <property name="lowerBound">
              <double>10.000000000000000</double>
             </property>
             <property name="upperBound">
              <double>-10.000000000000000</double>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbRaw">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <double>-10.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QwtSlider" name="sliderConverted">
             <property name="lowerBound">
              <double>0.000000000000000</double>
             </property>
             <property name="upperBound">
              <double>20000.000000000000000</double>
             </property>
             <property name="value">
              <double>10000.000000000000000</double>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbConverted">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>0</number>
             </property>
             <property name="maximum">
              <double>20000.000000000000000</double>
             </property>
             <property name="value">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Database</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="listDatabase"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="btnEnroll">
             <property name="text">
              <string>&amp;Enroll</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnDelete">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Remove</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnProperties">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Properties</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/images/properties.png</normaloff>:/images/properties.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnExport">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Export all templates</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>&amp;Import</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
